src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/base.yaml

```diff
@@ -29,12 +29,58 @@ twig_hooks:
         'sylius_shop.base.header.content':
             logo:
                 template: "@SyliusShop/shared/layout/base/header/content/logo.html.twig"
-            security_widget:
-                template: "@SyliusShop/shared/layout/base/header/content/security_widget.html.twig"
+            security:
+                template: "@SyliusShop/shared/layout/base/header/content/security.html.twig"
             cart:
                 component: 'sylius_shop:cart:widget'
             taxon_hamburger:
                 template: "@SyliusShop/shared/layout/base/header/content/taxon_hamburger.html.twig"
+                
+        'sylius_shop.base.header.content.security':
+            logged_in_user:
+                template: '@SyliusShop/shared/layout/base/header/content/security/logged_in_user.html.twig'
+            visitor:
+                template: '@SyliusShop/shared/layout/base/header/content/security/visitor.html.twig'
+                
+        'sylius_shop.base.header.content.security.logged_in_user':
+            mobile:
+                template: '@SyliusShop/shared/layout/base/header/content/security/logged_in_user/mobile.html.twig'
+            desktop:
+                template: '@SyliusShop/shared/layout/base/header/content/security/logged_in_user/desktop.html.twig'
+                
+        'sylius_shop.base.header.content.security.logged_in_user.mobile':
+            dropdown_button:
+                template: '@SyliusShop/shared/layout/base/header/content/security/logged_in_user/mobile/dropdown_button.html.twig'
```

Comment:
IMO we can put it in the dropdown template below, what's the point of having it separate?